src/components/atoms/HorizontalDivider.tsx

```diff
@@ -0,0 +1,73 @@
+import { cva } from "class-variance-authority";
+import { cn } from "../../utils/cn";
+import { Sizes } from "../../sharedVariants";
+
+const horizontalDividerVariants = cva("w-0.5 rounded-full", {
+  variants: {
+    color: {
```

Comment:
Can you move textColor, bgColor, and height to sharedVariants? From the sharedVariants folder, can you export a type from that file like `export type Color  = // shared colors` just to keep things consistent. Maybe check to see if there are other spots where this is worth doing

Reply:
done